prow/config/config.go

```diff
@@ -1003,6 +1008,14 @@ func parseProwConfig(c *Config) error {
 	}
 	c.GitHubOptions.LinkURL = linkURL
 
+	if c.StatusErrorLink == "" {
+		c.StatusErrorLink = "https://github.com/kubernetes/test-infra/issues"
+	}
+	_, err = url.Parse(c.StatusErrorLink)
```

Comment:
What purpose does this serve?

I don't really think this is necessary, and it will result in every prow component crashing if whenever this config value is bad.

Reply:
I guess since the prowJob status url is just a string, I don't think there would be any issues with displaying an invalid link, except that it wouldn't work if people clicked on it. This is mostly here to verify that the link is actually a valid url, reducing the chance of error.

I don't have a strong opinion here... should we keep it or loose it?

Reply:
Another option I just thought of would be to just log if it isn't valid and not error out? Would we prefer to do that?

Reply:
Prefer we just drop it in the config loading.

The most likely way people will discover this is by noticing the bad URLs when they click on it. This is more likely than someone scanning through logs and noticing this one, or noticing that they have a stale config because of this bad value.

If you want somewhere that validates the URL I would suggest putting that check into https://github.com/kubernetes/test-infra/tree/master/prow/cmd/checkconfig, which people can configure to run on presubmit.

Reply:
That makes sense. Take another look and let me know if that is what you were thinking.